config/services.php

```diff
@@ -2,41 +2,27 @@
 
 declare(strict_types=1);
 
-use Conticket\Conference\Domain\Repository\ConferenceRepositoryInterface;
-use Conticket\Conference\Factory\Repository\ConferenceRepositoryFactory;
-use Conticket\Conference\Factory\CommandHandler\CreateConferenceHandlerFactory;
-use Conticket\Conference\Domain\Command\CreateConference;
 use Conticket\Conference\Infrastructure\Service\CommandBusFactory;
 use Conticket\Conference\Infrastructure\Service\ConnectionFactory;
 use Conticket\Conference\Infrastructure\Service\EventStoreFactory;
+use Conticket\Conference\Infrastructure\Service\ApplicationFactory;
+use Conticket\Conference\Infrastructure\Service\RouterFactory;
 use Doctrine\DBAL\Connection;
-use Interop\Container\ContainerInterface;
 use Prooph\EventStore\EventStore;
 use Prooph\ServiceBus\CommandBus;
 use Zend\Expressive\Application;
 use Zend\Expressive\Router\FastRouteRouter;
 
+
 return (function () {
     return [
-        // @todo move factories to proper classes
         'factories' => [
-            Application::class     => function (ContainerInterface $container) {
-                return new Application($container->get(FastRouteRouter::class), $container);
-            },
-            FastRouteRouter::class => function (ContainerInterface $container) {
-                return new FastRouteRouter();
-            },
-
+            Application::class     => ApplicationFactory::class,
+            FastRouteRouter::class => RouterFactory::class,
```

Comment:
FastRouter doesn't need a specialised class for factory,  I guess you can use the `InvokableFactory::class` in this situations.